docs/abi.md

```diff
@@ -116,7 +116,7 @@ available on the channel, `CHANNEL_EMPTY` will be returned.
 - arg 5: Destination handle array count
 - arg 6: Address of a 4-byte location that will receive the number of handles
   retrieved with the message (as a little-endian u32)
-- return 0: Status of operation
+- return: Status of operation
```

Comment:
Note that Wasm is getting multiple return values at some point, this was for forward compatibility with that. https://hacks.mozilla.org/2019/11/multi-value-all-the-wasm/ 

I don't mind either way anyways, if you think it's confusing.

Reply:
Reverted (I was thinking that multiple returns are likely to far enough in the future that it's not worth allowing for them, but that link implies they might arrive sooner than I thought).